docs/UserGuide.md

```diff
@@ -82,13 +82,17 @@ Adds a patient to the patient record database.
 
 Format: `add n/NAME b/BIRTHDATE p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
+* Patient's name entered in is case-insensitive.
+* Duplicate patients are not allowed.
+  * Patients are considered duplicate when they have the same `NAME` and `ADDRESS`.
```

Comment:
I understand the execution flow for this check is that if the patient to be added has the same name and address as an existing patient in the address book, we throw a duplicate person exception. However, currently in the `AddCommand#execute`, the duplicate check is performed by `Model#hasPerson` which checks for equality through the `Person#isSamePerson` against each person in the address book, which validates that both persons do not have the same name.

To confirm, your PR seeks to change this sense of equality to checking for name (case-insensitive) AND address?

Reply:
Yes, this PR is to modify `Person#isSamePerson` to check for name (case-insensitive) and address.